### PR TITLE
Added cursorMove after line execution

### PR DESCRIPTION
**Pull request by Ladvien**

### CODE REVIEW

**Comment by Ikuyadeu** [comment]
@Ladvien Thank you for your Pull Request.
Looks solve #9, Very good!
Please waiting for my review.

**Comment by Ladvien** [comment]
No worries.  Let me know if it needs tweaks, if so I’ll edit and re-pull. 

And thank you for the extension—I use R a lot and VSCode is my favorite editor, so a big thank you.

—C. Thomas Brittain
Aka, Ladvien

File: src/extension.ts [hunk 1]
```diff
@@ -89,6 +89,9 @@ export function activate(context: ExtensionContext) {
         if (!rTerm) {
             createRterm(true);
         }
+        
+        commands.executeCommand('cursorMove', {'to':'down'});
```
Comment (Ikuyadeu) [inline src/extension.ts]: `{'to':'down'})` shoule be `{'to': 'down'})`.(Need one space)

**Comment by Ikuyadeu** [comment]
@Ladvien I looked your change and very useful!
My review content is not so important, if you don't feel like it.

**Comment by Ikuyadeu** [comment]
Very Good, Thank you!

**Pull request merged**